plugins/extra/p2js/componentEditors/P2BodyEditor.ts

```diff
@@ -61,6 +62,12 @@ export default class P2BodyEditor {
       this.editConfig("setProperty", "height", parseFloat(event.target.value));
     });
 
+    this.angleRow = SupClient.table.appendRow(this.tbody, SupClient.i18n.t("componentEditors:P2Body.angle"));
```

Comment:
Btw, I only added a translation for the english locale. Should I add a placeholder translation ("Angle"?) in the other locales to prevent it from showing as "componentEditors:P2Body.angle" in other languages? It kind of breaks the layout in the editor.